mocks/layouts.go

```diff
@@ -0,0 +1,37 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/influxdata/chronograf"
+)
+
+var _ chronograf.LayoutStore = &LayoutStore{}
+
+type LayoutStore struct {
+	AddF    func(ctx context.Context, layout chronograf.Layout) (chronograf.Layout, error)
+	AllF    func(ctx context.Context) ([]chronograf.Layout, error)
+	DeleteF func(ctx context.Context, layout chronograf.Layout) error
```

Comment:
lol, I didn't even notice that... just implemented what the compiler told me to 😛 . `LayoutStore` is the second-oldest store in this codebase, so I'm not entirely surprised there's some awkwardness.